src/lib/inspector.js

```diff
@@ -42,19 +45,17 @@ Inspector.prototype = {
 
     this.inspectorCameraEl = document.createElement('a-entity');
     this.inspectorCameraEl.isInspector = true;
-    this.inspectorCameraEl.addEventListener('loaded', function (entity) {
+    this.inspectorCameraEl.addEventListener('loaded', entity => {
```

Comment:
The rest of the code is already using arrow functions as it's the recommended for es6&react we just had this legacy code based on threejs' editor, so If we want to go back to named function we should discuss it and create another issue/PR to rewrite the rest of the code.